core/vm/interface.go

```diff
@@ -86,7 +86,7 @@ type StateDB interface {
 	Snapshot() int
 
 	AddLog(addr common.Address, topics []common.Hash, data []byte, blockNumber uint64)
-	GetLogData() [][]byte
+	GetLogData() ([][]common.Hash, [][]byte)
```

Comment:
nit: should we name parameters like other functions here?
`GetLogData() (topics [][]common.Hash, logData [][]byte)`

Reply:
added